src/app/filter/filter-fields.component.html

```diff
@@ -2,7 +2,7 @@
   <div class="input-group form-group">
     <div class="input-group-btn" dropdown>
       <button type="button" class="btn btn-default filter-fields dropdown-toggle" dropdownToggle
-              tooltip="Filter by" placement="{{config?.tooltipPlacement}}">
+              tooltip="Filter&nbsp;by" placement="{{config?.tooltipPlacement}}">
```

Comment:
I don't know that we want to prevent line wrapping for all. However, if you don't have enough room for the tooltip, you could add your own CSS. That is, using Adam's suggestion or by setting width.

See the example given here:
https://github.com/valor-software/ngx-bootstrap/issues/808